provisioning/roles.txt

```diff
@@ -5,3 +5,4 @@ geerlingguy.mysql,1.5.0
 Mayeu.RabbitMQ,1.4.0
 nodesource.node
 Stouts.elasticsearch,2.5.0
+gunicorn,19.3
```

Comment:
This isn't the right file for Python dependencies. Add this to the file `/provisioning/roles/kuma/tasks/python.yml` after the installation of `performance.txt` requirements file:

```yml
- name: Install some more deps in the virtualenv
  pip:
    virtualenv: "/home/vagrant/env"
    virtualenv_site_packages: yes
    name: "{{ item }}"
  with_items:
    - gunicorn
```